spec/support/system_description_factory.rb

```diff
@@ -810,6 +916,10 @@ def build_description(name, store, options)
           "name": "sshd.service",
           "state": "enabled"
         },
+        {
+          "name": "NetworkManager.service",
+          "state": "enabled"
+        },	
```

Comment:
You can not just extend an example, unless you adapt all the others tests which are based on it.

For the unit tests it is ok, if some services are missing from the kickstart manifest, as long as the result is consistent.